plugins/org.eclipse.n4js.model/emf-gen/org/eclipse/n4js/n4JS/IndexedAccessExpression.java

```diff
@@ -28,7 +28,7 @@
  * @model
  * @generated
  */
-public interface IndexedAccessExpression extends Expression {
+public interface IndexedAccessExpression extends Expression, MemberAccess {
```

Comment:
excellent find!!

But: we cannot solve this for all cases, because we have a conflict of goals here. We cannot say:
* I want the scoping to return members that are contained in a resource AND
* I want to invoke getAllElements() on the scope AND
* I don't want the composed member cache to explode.

There's no way to solve all of that simultaneously.

But I'll try to improve the client code handling index expressions to avoid calling #getAllElements() + I'll update API doc.